cypress/e2e/the-order-of-markings.cy.js

```diff
@@ -0,0 +1,14 @@
+describe("will check the order of markings are in the right order as we typed them in the text box", () => {
+    beforeEach(() => {
+        cy.visit("/");
+    });
+
+    it("will check the marking order as written on text box", () => {
+        cy.get('[data-test="editor"]').type("lider ");
+        cy.get("span.loanword").contains("lider").should("exist");
+        cy.get('[data-test="editor"]').type("gabmim ");
+        cy.get("span.typo").contains("gabmim").should("exist");
+        cy.get('[data-test="editor"]').type("Shkoi tek zyra.");
+        cy.get("span.typo").contains("tek").should("exist");
```

Comment:
Hmm unsure if the order is tested like this, i.e. even writing `cy.get("span.typo").contains("lider").should("exist");` on line 10 would still pass and nothing regarding the order would be tested, I think.

I'd suggest writing something like:
1. write "gabmim lider" in the editor
2. use `eq` to pick the first and second children (0 and 1) from the markings and check whether the first one belongs to "gabmim" and the second to "lider"